<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget_Test">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>171</height>
    </rect>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::NoDragDrop</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>1st Item</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2nd Item</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3rd Item</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>381</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="line_insertItem">
      <property name="text">
       <string>Insert Item</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spin_insertRow"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_insertItem">
      <property name="text">
       <string>Insert</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>381</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="line_addItem">
      <property name="text">
       <string>Add Item</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_addItem">
      <property name="text">
       <string>add</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="btn_printItem">
      <property name="text">
       <string>Print</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_printMultiItems">
      <property name="text">
       <string>Print_Multi</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_removeItem">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_clearItem">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
